sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -36,18 +36,6 @@
 from .. import tekton_api_version
 
 
-def _literal_str_representer(dumper, data):
```

Comment:
I forgot the history of this, can you quickly explain why we had to add this and why it's no longer needed?

Reply:
I wanted to add literal style to make the script part looks nicer. But there's some edge cases with pyyaml when using it on integer since it's designed with yaml 1.1

Here's the definition of literal style in yaml
https://yaml.org/spec/1.2/spec.html#id2795688